tests/python/pants_test/backend/jvm/tasks/jvm_compile/java/test_java_compile_settings_partitioning.py

```diff
@@ -40,8 +40,8 @@ def _version(self, version):
     return Revision.lenient(version)
 
   def _task_setup(self, targets, platforms=None, default_platform=None, **options):
-    options['source'] = options.get('source', '1.7')
-    options['target'] = options.get('target', '1.7')
+    options['source'] = options.get('source', '13')
```

Comment:
NB: I intentionally changed these to use `13` instead of `1.13` because JVM dropped the leading `1.` in version strings starting with Java 9.